src/version.h

```diff
@@ -4,8 +4,8 @@
  * similar. */
 #define SERVER_NAME "valkey"
 #define SERVER_TITLE "Valkey"
-#define VALKEY_VERSION "255.255.255"
-#define VALKEY_VERSION_NUM 0x00ffffff
+#define VALKEY_VERSION "7.255.0"
```

Comment:
In Redis, version 7.4-rc1 defines as 
#define REDIS_VERSION "7.3.240"
#define REDIS_VERSION_NUM 0x000703f0

version 7.0-rc1 defines as 
#define REDIS_VERSION "6.9.240"
#define REDIS_VERSION_NUM 0x000609f0

Thus, here I think we should define 8.0 RC1 as

#define REDIS_VERSION "7.9.240"
#define REDIS_VERSION_NUM 0x000709f0

Reply:
For some reason I thought we used 0 in the patch, I think 240 is fine.